MAUI/Autocomplete/Searching-Filtering.md

```diff
@@ -0,0 +1,176 @@
+
+<editors:SfAutocomplete x:Name="autocomplete"
+                        WidthRequest="250" 
+                        ItemsSource="{Binding SocialMedias}"
+                        DisplayMemberPath="Name" />
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+autocomplete.DisplayMemberPath = "Name";
+
+{% endhighlight %}
+{% endtabs %}
+
+For e.g. After typing `T` in selection box, social medias which have starting letter `T` will be listed in drop-down.
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete search based on display member path](Images/Searching_Filtering/SearchBasedDisplayMemberPath.png)
+
+### Searching based on TextMemberPath
+
+Searching will be performed based on the `TextMemberPath` property, while entering the text into the selection box. If `TextMemberPath` is `null` or `string.Empty`, searching will be performed based on `DisplayMemberPath`. 
+
+
+{% tabs %}
+{% highlight xaml %}
+
+<editors:SfAutocomplete x:Name="autocomplete"
+                        WidthRequest="250"
+                        ItemsSource="{Binding SocialMedias}"
+                        TextMemberPath="ID"
+                        DisplayMemberPath="Name" />
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+autocomplete.TextMemberPath = "ID";
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete search based on text member path](Images/Searching_Filtering/SearchBasedTextMemberPath.png)
+
+## Filtering mode
+
+The `TextSearchMode` property of the `Autocomplete` can be used to regulate how the control behaves when it receives user input. The default text filtering type is `StartsWith`, which ignores accent and is case insensitive. The text filter modes that are available are as follows,
+
+* StartsWith
+* Contains
+
+### Filter with beginning text
+
+Set the `TextSearchMode` property value to `StartWith` to filter matching items based on the starting text. The first item in the drop-down list that matches the user input will be highlighted. 
+
+{% tabs %}
+{% highlight XAML %}
+
+<editors:SfAutocomplete x:Name="autoComplete"
+                        WidthRequest="250"
+                        TextSearchMode="StartsWith"
+                        ItemsSource="{Binding SocialMedias}"
+                        DisplayMemberPath="Name"
+                        TextMemberPath="Name" />
+
+{% endhighlight %}
+
+{% highlight C# %}
+
+autocomplete.TextSearchMode = AutocompleteTextSearchMode.StartsWith;
+
+{% endhighlight %}
+{% endtabs %}
+
+The following image illustrates the result of the above code:
+
+![.NET MAUI Autocomplete filtering based on started text](Images/Searching_Filtering/SearchModeStartsWith.png)
+
+### Filter with contains text
+
+Set the `TextSearchMode` property value to `Contains` to filter the matching items based on the containing specific text. The first item  in the drop-down list that matches the user input will be highlighted. 
```

Comment:
Set the `TextSearchMode` property value to `Contains` to filter the matching items based on the containing specific text. The first item in the drop-down list that matches the user input will be highlighted.